azure/activity_logs_monitoring/index.js

```diff
@@ -11,6 +11,7 @@ const STRING = 'string'; // example: 'some message'
 const STRING_ARRAY = 'string-array'; // example: ['one message', 'two message', ...]
 const JSON_OBJECT = 'json-object'; // example: {"key": "value"}
 const JSON_ARRAY = 'json-array'; // example: [{"key": "value"}, {"key": "value"}, ...] or [{"records": [{}, {}, ...]}, {"records": [{}, {}, ...]}, ...]
+const BUFFER_ARRAY = 'buffer-array'; // example: <Buffer 7b 22 72 65 63 6f 72 64 73 22 3a 20 5b 7b 20 22 74 69 6d 65 22 3a 20 22 74 65 73 74 69 6e 67 22 7d 5d 7d>
```

Comment:
This would technically be in an array right? I think we could shorten this to something like `[<Buffer obj>, <Buffer obj>]`.